app/entry.server.tsx

```diff
@@ -89,7 +80,6 @@ export default async function handleRequest(
       "ws://127.0.0.1:*",
       "https://api.okendo.io",
       "https://cdn-static.okendo.io",
-      "https://s0gpgg13u6.execute-api.ap-southeast-2.amazonaws.com",
       "https://surveys.okendo.io",
       "https://5i27ysv3j8.execute-api.us-west-2.amazonaws.com",
```

Comment:
Probably worth double checking this and some of the other cloudfront.net urls we have in here, better if we can call a helper script or function that will patch the CSP from our library